Formula/commandbox.rb

```diff
@@ -19,7 +19,7 @@ def install
   end
 
   test do
-    system "box", "--commandbox_home=~/", "version"
-    system "box", "--commandbox_home=~/", "help"
+    system "/usr/local/bin/box", "--commandbox_home=~/", "version"
+    system "/usr/local/bin/box", "--commandbox_home=~/", "help"
```

Comment:
Sorry guys, I misinterpreted the message from the Jenkins build that I needed to put in the actual path to the binary.  I didn't realize that I actually needed the text `#{bin}`  I'll update now.